src/http/kv/mod.rs

```diff
@@ -0,0 +1,93 @@
+mod create_namespace;
+
+pub use create_namespace::create_namespace;
+
+use cloudflare::auth::Credentials;
+use cloudflare::response::{APIError, APISuccess};
+use cloudflare::workerskv;
+use cloudflare::HTTPAPIClient;
+
+use crate::settings::global_user::GlobalUser;
+
+impl From<GlobalUser> for Credentials {
+    fn from(user: GlobalUser) -> Credentials {
+        Credentials::User {
+            key: user.api_key,
+            email: user.email,
+        }
+    }
+}
+
+fn new_api_client(user: GlobalUser) -> HTTPAPIClient {
+    let credentials = Credentials::from(user);
+
+    HTTPAPIClient::new(credentials)
+}
+
+#[derive(Debug)]
+pub struct KvNamespace {
+    id: String,
+    title: String,
+}
+
+impl From<APISuccess<workerskv::WorkersKVNamespace>> for KvNamespace {
+    fn from(success: APISuccess<workerskv::WorkersKVNamespace>) -> KvNamespace {
+        KvNamespace {
+            id: success.result.id,
+            title: success.result.title,
+        }
+    }
+}
+
+// TODO: Map these to Wrangler-relevant messages? or an Enum?
+fn api_errors_to_message(errors: Vec<APIError>) -> String {
+    let mut messages = Vec::new();
+    for error in errors {
+        match error.code {
+            // https://api.cloudflare.com/#workers-kv-namespace-errors
```

Comment:
these error messages are literally just mapped from the linked docs; technically these are exactly the same text as we would pull out of the `message` part of the error response, but here is where we could translate them to wrangler-specific messages, including actionable feedback to the user